providers/dns/httpreq/httpreq.go

```diff
@@ -69,8 +69,8 @@ func NewDNSProvider() (*DNSProvider, error) {
 
 	config := NewDefaultConfig()
 	config.Mode = os.Getenv("HTTPREQ_MODE")
```

Comment:
I think you can use `GetOrFile` for the mode.

Reply:
Right, I didn't realize the other vars get it from `_FILE` as well indirectly via `env.Get` and friends.  cde25a5 should now support reading from file for all `HTTPREQ_` env vars.